docs/compatibility.md

```diff
@@ -865,3 +865,10 @@ Seq(0L, Long.MaxValue).toDF("val")
 
 But this is not something that can be done generically and requires inner knowledge about
 what can trigger a side effect.
+
+## HyperLogLogPlusPlus(approx_count_distinct)
+The maximum supported precision is 15. The formula of precision is:
```

Comment:
Can you clarify about the maximum precision supported in Spark? If they are different, then why there is such difference?